pagan/pagan.py

```diff
@@ -50,12 +50,15 @@ def save(self, path=DEFAULT_OUTPUT_PATH, filename=DEFAULT_FILENAME):
             os.makedirs(path)
 
         # Cut the .png file ending if one was omitted.
-        if filename[-4:] == ".png":
+        if !str.startswith(extensionT, ".")
+        	extension = str.subtext(exstension, 1 - len(exstension)); 
```

Comment:
if you wanted to make a difference between this 'extension' here and the above - i'd go for final_extension or something like that.